client/src/utils/jobsUtil.js

```diff
@@ -61,69 +71,85 @@ export function getJobDisplayInformation(job, app) {
     // Improve any values with app information
     try {
       try {
-        display.systemName = getSystemName(app.exec_sys.login.host);
+        display.systemName = getSystemName(app.exec_sys.host);
       } catch (ignore) {
         // ignore if there is problem improving the system name
       }
 
-      display.applicationName = app.definition.label;
+      // TODO V3: Undefined for now
+      // display.applicationName = app.definition.notes.label;
+
+      // TODO V3: Unsure if needed
 
       // Improve input/parameters
-      display.inputs.forEach((input) => {
-        const matchingParameter = app.definition.inputs.find((obj) => {
-          return input.id === obj.id;
-        });
-        if (matchingParameter) {
-          // eslint-disable-next-line no-param-reassign
-          input.label = matchingParameter.details.label;
-        }
-      });
-      display.parameters.forEach((input) => {
-        const matchingParameter = app.definition.parameters.find((obj) => {
-          return input.id === obj.id;
-        });
-        if (matchingParameter) {
-          // eslint-disable-next-line no-param-reassign
-          input.label = matchingParameter.details.label;
-        }
-      });
+      // display.inputs.forEach((input) => {
+      //   const matchingParameter = app.definition.jobAttributes.fileInputs.find((obj) => {
+      //     return input.id === obj.name;
+      //   });
+      //   if (matchingParameter) {
+      //     // eslint-disable-next-line no-param-reassign
+      //     input.label = matchingParameter.details.label;
+      //   }
+      // });
+      // display.parameters.forEach((input) => {
+      //   const matchingParameter = app.definition.parameters.find((obj) => {
+      //     return input.id === obj.id;
+      //   });
+      //   if (matchingParameter) {
+      //     // eslint-disable-next-line no-param-reassign
+      //     input.label = matchingParameter.details.label;
+      //   }
+      // });
+
+      // TODO V3: Maybe should filter with includes? some have null/array values
```

Comment:
We'll probably have to filter with a flag we create ourselves with whatever meta object they allow us to attach to job input args in the future. For example, a webhookUrl will be a required input for interactive jobs, but we want to hide that input